openstates/me/people.py

```diff
@@ -119,30 +143,30 @@ def scrape_reps(self):
 
```

Comment:
Looks like every legislator in this spreadsheet has an address. I'm fine if you chop this "default address" section out, and just don't assign an address to the legislator if it isn't present.